packages/phone-number-privacy/src/config.ts

```diff
@@ -1,4 +1,10 @@
+import dotenv from 'dotenv'
 const functions = require('firebase-functions')
-export const config = functions.config()
+export const functionConfig = functions.config()
 
-console.log(config)
+// Load environment variables from .env file
+dotenv.config()
+
+export const PHONE_NUMBER_PRIVACY_SECRET_KEY =
+  (process.env.SECRET_KEY as string) || functionConfig.envs.secret_key //TODO @aslawson what is best way to include private key to GCP?
```

Comment:
We can chat about this in our next meeting. For now, hold off on putting the real key into firebase config